pallets/asset-manager/src/lib.rs

```diff
@@ -532,7 +536,8 @@ pub mod pallet {
 				// Send tokens back to relaychain.
 				Junctions::X1(Junction::AccountId32 { .. }) => true,
 				// Send tokens to sibling chain.
-				Junctions::X2(Junction::Parachain(para_id), Junction::AccountId32 { .. }) => {
+				Junctions::X2(Junction::Parachain(para_id), Junction::AccountId32 { .. })
+				| Junctions::X2(Junction::Parachain(para_id), Junction::AccountKey20 { .. }) => {
```

Comment:
We'll probably end up needing to support more junctions than `AccountKey20` like `AccountIndex64` . But I guess let's cross that bridge when we get there.

Reply:
For now, I didn't see any use case on `AccountIndex64`.